force-app/main/default/classes/SOQL.cls

```diff
@@ -39,9 +41,11 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Contact> contactsByFirstName () {
+        String nameToFind = 'John';
+        List<Contact> listOFContacts = [SELECT Name, Id FROM Contact WHERE FirstName = :nameToFind];
+        Return listOfContacts;
+    }
```

Comment:
Can you build this method to take `nameToFind` as an argument so that it is more useful?